tests/SkiaSharp.Tests.Devices/Tests/iOS/iOSExtensionsTests.cs

```diff
@@ -99,5 +101,21 @@ public void CGRectToSKRect(int x, int y, int w, int h)
 
 			Assert.Equal(expected, actual);
 		}
+
+		[SkippableFact]
+		public void GRContextDisposeDoesNotCrash()
```

Comment:
For this test, we probably can do something linke this:

```cs
using var metalView = new SKMetalView();
((IMTKViewDelegate)metalView).Draw(metalView);
```

I know this is not as nice as what you have here, but for now let's keep this contained until we figure out the rules.